tools/test-proxy/Azure.Sdk.Tools.TestProxy.Tests/Test.RecordEntries/sample_entry.json

```
@@ -0,0 +1,361 @@
+        "Authorization": "Sanitized",
+        "Content-Length": "0",
+        "User-Agent": "azsdk-java-azure-communication-phonenumbers/1.2.0-beta.1 (17.0.1; Windows 11; 10.0)",
+        "x-ms-client-request-id": "466a624f-e5fc-48ea-a837-a786f7fe98de",
+        "x-ms-content-sha256": "47DEQpj8HBSa\u002B/TImW\u002B5JCeuQeRkm5NMpJWZG3hSuFU=",
+        "x-ms-date": "Fri, 14 Jul 2023 22:02:01 GMT",
+        "x-ms-hmac-string-to-sign-base64": "R0VUCi9zaXA/YXBpLXZlcnNpb249MjAyMy0wMy0wMQpGcmksIDE0IEp1bCAyMDIzIDIyOjAyOjAxIEdNVDtsaXZlLXRlc3RpbmcuY29tbXVuaWNhdGlvbi5henVyZS5jb207NDdERVFwajhIQlNhKy9USW1XKzVKQ2V1UWVSa201Tk1wSldaRzNoU3VGVT0="
+      },
+      "RequestBody": null,
+      "StatusCode": 200,
+      "ResponseHeaders": {
+        "api-deprecated-versions": "2021-05-01-preview, 2022-09-01-preview, 2023-01-01-preview",
+        "api-supported-versions": "2023-03-01, 2023-04-01-preview",
+        "Content-Length": "246",
+        "Content-Type": "application/json; charset=utf-8",
+        "Date": "Fri, 14 Jul 2023 22:02:02 GMT",
+        "MS-CV": "NWlZ4Hs99US2eoltIWSOxQ.0",
+        "Strict-Transport-Security": "max-age=2592000",
+        "X-Azure-Ref": "02sWxZAAAAACWspOFjMttS5eUTqb4rP0SV1NURURHRTA4MTQAOWZjN2I1MTktYThjYy00Zjg5LTkzNWUtYzkxNDhhZTA5ZTgx",
+        "X-Cache": "CONFIG_NOCACHE",
+        "X-Processing-Time": "696ms"
+      },
+      "ResponseBody": {
+        "trunks": {},
+        "routes": [
+          {
+            "description": "desc0",
+            "name": "route0",
+            "numberPattern": "0.*",
+            "trunks": []
+          },
+          {
+            "description": "desc1",
+            "name": "route1",
+            "numberPattern": "1.*",
+            "trunks": []
+          },
+          {
+            "description": "desc2",
+            "name": "route2",
+            "numberPattern": "2.*",
+            "trunks": []
+          }
+        ]
+      }
+    },
+    {
+      "RequestUri": "https://REDACTED/sip?api-version=2023-03-01",
+      "RequestMethod": "PATCH",
+      "RequestHeaders": {
+        "Accept": "application/json",
+        "Authorization": "Sanitized",
+        "Content-Length": "352",
+        "Content-Type": "application/merge-patch\u002Bjson",
+        "User-Agent": "azsdk-java-azure-communication-phonenumbers/1.2.0-beta.1 (17.0.1; Windows 11; 10.0)",
+        "x-ms-client-request-id": "3cc2a1c9-7df4-4992-be65-31249cf19a8f",
+        "x-ms-content-sha256": "mRg5okj8G4rBquzMpPC4\u002BU9U9ZPQmzEvQLHi33eBv\u002BE=",
+        "x-ms-date": "Fri, 14 Jul 2023 22:02:02 GMT",
+        "x-ms-hmac-string-to-sign-base64": "UEFUQ0gKL3NpcD9hcGktdmVyc2lvbj0yMDIzLTAzLTAxCkZyaSwgMTQgSnVsIDIwMjMgMjI6MDI6MDIgR01UO2xpdmUtdGVzdGluZy5jb21tdW5pY2F0aW9uLmF6dXJlLmNvbTttUmc1b2tqOEc0ckJxdXpNcFBDNCtVOVU5WlBRbXpFdlFMSGkzM2VCditFPQ=="
+      },
+      "RequestBody": {
+        "trunks": {
+          "third-69452be928bv46ba85871d092a0676e9.992a0be8-d789-4628-af1e-73e7d2c57bed.abs.test": {
```

Hey @scbedd would it even be possible to redacte this ```suggestion
          "third-69452be928bv46ba85871d092a0676e9.992a0be8-d789-4628-af1e-73e7d2c57bed.abs.test": {
```
to

```
          "third-69452be928bv46ba85871d092a0676e9.redacted.com": {
```

I know this regex works as is
https://regex101.com/r/cnt4ms/1